core/src/config.jl

```diff
@@ -97,6 +98,19 @@ end
     compression_level::Int = 6
 end
 
+function Base.convert(::Type{LogLevel}, level::AbstractString)
```

Comment:
Can we not avoid piracy by overloading `Configurations.from_dict`?

https://configurations.rogerluo.dev/stable/convert/